snyk/cli
# refactor: Relocated new IaC output tests

#### What does this PR do?

- Relocated tests for IaC output to match the source files hierarchy.

#### Where should the reviewer start?
```
test/jest/unit/lib/formatters/iac-output/v2
```

#### How should this be manually tested?
- Ensure the moved tests all pass.
- Ensure the moved tests match the source files hierarchy.
